tensorflow/tools/tf_sig_build_dockerfiles/setup.rocm.sh

```diff
@@ -139,7 +139,7 @@ fi
 function ver { printf "%03d%03d%03d" $(echo "$1" | tr '.' ' '); }
 if [ $(ver "$ROCM_VERSION") -lt $(ver "6.2.0") ]
 then
-  echo "build:rocm --copt=-fclang-abi-compat=17" >> /etc/bazel.bazelrc
+  echo "build:rocm_base --copt=-fclang-abi-compat=17" >> /etc/bazel.bazelrc
```

Comment:
could you add a comment to explain this for resolving linker issue in llvm17, please?